app/src/main/resources/application.yml

```diff
@@ -7,7 +7,169 @@ jetty:
     maxThreads: 200
     minThreads: 8
     idleTimeout: 60000
+
+---
+spring:
+  config:
+    activate:
+      on-profile: dev
+  datasource:
+    url: jdbc:postgresql://sharedaurorapsql-epdcs-dev-use1.cluster-chxkh4p9th7p.us-east-1.rds.amazonaws.com:5432/nakadi
+    username: nakadi_user
+    driverClassName: org.postgresql.Driver
+    tomcat:
+      initial-size: 2
+      max-active: 2
+      min-idle: 1
+      max-idle: 1
+      test-on-borrow: true
+      validation-query: SELECT 1
+      test-while-idle: true
+      time-between-eviction-runs-millis: 5000
+      min-evictable-idle-time-millis: 60000
+      connection-properties: socketTimeout=10;connectTimeout=10;loginTimeout=5
+
+nakadi:
+  admin.default:
+    dataType: service
+    value: stups_nakadi
+  authz.warnAllDataAccessMessage: "Data access warning"
+  eventType.deletableSubscription:
+    owningApplication: "nakadi_archiver"
+    consumerGroup: "nakadi_to_s3"
+  features.defaultFeatures:
+    DISABLE_EVENT_TYPE_CREATION: false
+    DISABLE_EVENT_TYPE_DELETION: false
+    DISABLE_SUBSCRIPTION_CREATION: false
+    REMOTE_TOKENINFO: true
+    KPI_COLLECTION: true
+    DISABLE_DB_WRITE_OPERATIONS: false
+    FORCE_EVENT_TYPE_AUTHZ: false
+    FORCE_SUBSCRIPTION_AUTHZ: false
+    EVENT_OWNER_SELECTOR_AUTHZ: false
+    ACCESS_LOG_ENABLED: true
+  topic:
+    min:
+      retentionMs: 10800000 # 3 hours
+    max:
+      partitionNum: 8
+      retentionMs: 345600000 # 4 days
+    default:
+      partitionNum: 1
+      replicaFactor: 2
+      retentionMs: 172800000 # 2 days
+      rotationMs: 10800000 # 3 hours
+    compacted:
+      rotationMs: 10800000 # 3 hours
+      segmentBytes: 1073741824 # 1 GB
+      compactionLagMs: 10800000 # 3 hours
+      warnMessage: "Compaction warning"
+  stream:
+    timeoutMs: 31536000000 # 1 year :-P
+    max.commitTimeout: 60 # 1 minute
+    maxConnections: 5
+    maxStreamMemoryBytes: 50000000 # ~50 MB
+  kafka:
+    bootstrap.servers: b-2.epdcs-nakadi-msk-kafk.uwqhdz.c23.kafka.us-east-1.amazonaws.com:9098,b-3.epdcs-nakadi-msk-kafk.uwqhdz.c23.kafka.us-east-1.amazonaws.com:9098,b-1.epdcs-nakadi-msk-kafk.uwqhdz.c23.kafka.us-east-1.amazonaws.com:9098
+    min.insync.replicas: 2
+    retries: 5
+    request.timeout.ms: 50000
+    instanceType: t2.large
```

Comment:
is this the instance type of the kafka instance we are using? we should verify. also check what this conig is being used for in the code

Reply:
There are statistic for relevant instance type in 'partitions_statistics.json' file. According to the stats the partitions count is calculated.  95fece51b0a8a44784aa3793c6faa7b9e0946377